cltk/phonology/middle_english/transcription.py

```diff
@@ -0,0 +1,104 @@
+__author__ = ['Chatziargyriou Eleftheria <ele.hatzy@gmail.com>']
+__license__ = 'MIT License'
+"""
+The hyphenation/syllabification algorithm is based on the typical syllable structure model of onset/nucleus/coda.
+"""
+
+
+"""
+An additional problem arises with the distinction between long and short vowels, since many use identical graphemes for
+both long and short vowels. The great vowel shift that dates back to the early stages of ME poses an additional problem
+"""
+SHORT_VOWELS = ['a', 'e', 'i', 'o', 'u', 'y', 'æ']
+
+LONG_VOWELS = ['aa', 'ee', 'oo', 'ou', 'ow', 'ae']
+
+DIPHTHONGS = ['th', 'gh', 'ht', 'ch']
+
+TRIPHTHONGS = ['ght']
+
+CONSONANTS = ['b', 'c', 'd', 'f', 'g', 'h', 'l', 'm', 'n', 'p', 'r', 's', 't', 'x', 'ð', 'þ', 'ƿ']
+
+class Word:
+
+    def __init__(self, word):
+        self.word = word
+        self.syllabified = ''
+
+    def syllabify(self):
+
+        #Array holding the index of each given syllable
+        ind = []
+
+        i = 0
+        #Iterate through letters of word searching for the nuclei
+        while i < len(self.word) - 1:
+
+            if self.word[i] in SHORT_VOWELS:
+
+                nucleus = ''
+
+                #Find cluster of vowels
+                while self.word[i] in SHORT_VOWELS and i < len(self.word) - 1:
+                    nucleus += self.word[i]
+                    i += 1
+
+                try:
+                    #Check whether it is suceeded by a geminant
+
+                    # Throught the early 11th-14th century, ME went through a process of loss of gemination (double
+                    # consonants. Originally, the syllable preceding a geminate was a closed one. The following assumes
+                    # any occuring geminates will be separated like in Modern English (working both as coda of first
+                    # syllable and onset of the other
```

Comment:
Please place these comments in docstring comment at the function level.
Elsewhere use the Sphinx inline doc comment marker "#:" to flag comments you would like maybe include in detailed function level documentation.
(e.g. see: http://www.sphinx-doc.org/en/master/ext/example_google.html  re: the line:
self.attr3 = param3  #: Doc comment *inline* with attribute 
)